joern-cli/build.sbt

```diff
@@ -117,22 +117,20 @@ generateScaladocs := {
   out
 }
 
-lazy val packageZip = taskKey[File]("create zip with joern-cli, including schema-extender")
-packageZip := {
-  import org.zeroturnaround.zip.ZipUtil
+/* add schemas from codepropertygraph and schema-extender */
+Universal/packageBin/mappings ++= {
+  import better.files.File
   val joernCliStaged = (Universal/stage).value
-  val schemaExtenderStaged = (Projects.schemaExtender/Universal/stage).value
-  val schemaExtenderInstallLocation = joernCliStaged/"schema-extender"
-
-  IO.copyDirectory(schemaExtenderStaged, schemaExtenderInstallLocation, CopyOptions(overwrite = true, preserveLastModified = true, preserveExecutable = true))
+  val schemaExtenderStaged = (Projects.schemaExtender/Universal/stage).value.toPath
+  val tmpDir = File.newTemporaryDirectory("joern-cli-build").deleteOnExit
```

Comment:
Unless I'm missing something, the `joern-cli-build` prefix is not used anywhere, so might as well not specify it

Reply:
it's just a prefix for the temporary dir created in /tmp. it's not necessary, but good practice to do so: just in case something goes wrong and you want to know where that random file came from, it gives you a hint where to start looking